components/Decks.jsx

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { View, Text, FlatList, StyleSheet } from "react-native";
+
+const Decks = () => {
+  const testDecks = [
+    { name: "JS", description: "Testtesttest", id: 1 },
+    { name: "Python", description: "Testtesttest", id: 2 },
+    { name: "HTML", description: "Testtesttest", id: 3 },
+  ];
+  const [currentDecks, setCurrentDecks] = useState([]);
+
+  useEffect(() => {}, []);
+
+  // const renderItem = ({ item }) => {
+  //   return (
+  //     <View style={deckStyles.deckList}>
+  //       <Text style={deckStyles.name}>{item.name}</Text>
+  //       <Text style={deckStyles.description}>{item.description}</Text>
+  //     </View>
+  //   );
+  // };
```

Comment:
Why not use render item with Flatlist below ?

Reply:
@reemhd just clarified me on the reason all good though ! 😄  I think the error of flatList needing a parent tag could be reoslved with https://reactnative.dev/docs/flatlist?language=javascript They use safeView there 👍